var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -137,9 +158,18 @@ def _def(variant, flag=None):
 
         if '+cuda' in spec:
             cuda_arch = spec.variants['cuda_arch'].value
-            if cuda_arch:
+            if cuda_arch[0] != 'none':
                 args += [
                     '-DCMAKE_CUDA_FLAGS=-arch=sm_{0}'.format(cuda_arch[0])
                 ]
 
         return args
+
+    def setup_environment(self, spack_env, run_env):
```

Comment:
If you add `extends('python', when='+python')` above, you won't need this function.

Reply:
Good to know. Thanks!